hack/find-rules.sh

```diff
@@ -3,8 +3,10 @@ set -e
 
 # find all rule groups and aggregate them
 # returns a json list of groups
-
-find assets/ -type f -name "*prometheus-rule.yaml" | while read -r f
-do
-    tmp/bin/gojsontoyaml -yamltojson < "$f" | jq .spec
+find assets/ -type f -name "*.yaml" | while read -r f; do
+    KIND=$(tmp/bin/gojsontoyaml -yamltojson <"$f" | jq .kind)
```

Comment:
instead of converting the file twice from yaml to json, let's store the output in a variable and then extract the kind / spec